### PR TITLE
feat(replay): Implement a basic a11y table for replay details

### PR DESCRIPTION
**Pull request by ryan953**
Here's the new table, with mock data, sorted by timestamp: 

SCR-20231012-jttt
SCR-20231012-juao

### CODE REVIEW

File: static/app/views/replays/detail/accessibility/index.tsx
Comment (ryan953) [inline static/app/views/replays/detail/accessibility/index.tsx:1]: I left some commented code in here to activate in the followup PRs

File: static/app/views/replays/detail/accessibility/accessibilityTableCell.tsx [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+        </Cell>
+      ),
+      () => (
+        <Cell {...columnProps}>
+          <Text>{a11yIssue.id ?? EMPTY_CELL}</Text>
+        </Cell>
+      ),
+      () => (
+        <Cell {...columnProps}>
+          <Text>{a11yIssue.elements?.[0].element ?? EMPTY_CELL}</Text>
```
Comment (billyvg) [inline static/app/views/replays/detail/accessibility/accessibilityTableCell.tsx:124]: Is there a `<code>`-type component we can use here?

File: static/app/views/replays/detail/accessibility/index.tsx [hunk 2]
```diff
@@ -1,13 +1,272 @@
...
+  border: 1px solid ${p => p.theme.border};
+  border-radius: ${p => p.theme.borderRadius};
+
+  .beforeHoverTime + .afterHoverTime:before {
+    border-top: 1px solid ${p => p.theme.purple200};
+    content: '';
+    left: 0;
+    position: absolute;
+    top: 0;
+    width: 999999999%;
```
Comment (billyvg) [inline static/app/views/replays/detail/accessibility/index.tsx:241]: Does this behave differently than `100%`?

**Review by billyvg — APPROVED** [review]

File: static/app/views/replays/detail/accessibility/index.tsx (hunk 2 above)
Comment (ryan953) [reply static/app/views/replays/detail/accessibility/index.tsx:241]: yes! 100% would be the width of the column, this is set to be really/arbitrarily wide so it crosses under all columns, starting at the left edge of the table (left: 0, the relative parent is the container of the table)

It's fully copied from network and errors tabs, this is the 3rd example which means time to refactor in a followup. There are more things to refactor too

File: static/app/views/replays/detail/accessibility/accessibilityTableCell.tsx (hunk 1 above)
Comment (ryan953) [reply static/app/views/replays/detail/accessibility/accessibilityTableCell.tsx:124]: oh yeah

SCR-20231012-jttt

**Pull request merged**